cmd/src/server.rs

```diff
@@ -281,6 +276,17 @@ impl TiKVServer {
         .unwrap();
     }
 
+    fn init_yatp(&self) {
+        let yatp_enabled = self.config.readpool.unify_read_pool;
```

Comment:
Why not
```rust
        if !self.config.readpool.unify_read_pool {
            return;
        }
```

Reply:
I just want to leave a hint that if there are other options which will enable yatp in the future, the options should be added here.